CONTRIBUTING.md

```diff
@@ -62,7 +62,16 @@ included in the project:
    git checkout -b <topic-branch-name>
    ```
 
-4. Install the [Jekyll](https://github.com/jekyll/jekyll/) gem to preview locally.
+4. Install the [Jekyll](https://github.com/jekyll/jekyll/) gem and dependencies to preview locally:
+
+    ```bash
+    # Install the needed gems
+    gem install jekyll jekyll-sitemap pygments.rb
```

Comment:
Yeah I added that gem to a new gemfile yesterday, so updating the contributor instructions to mention bundler would be great. 

--
Phil Sturgeon
Sent from my iPhone and there's probably typos because I'm probably at the pub.

> On Feb 15, 2016, at 12:14 AM, Juliette <notifications@github.com> wrote:
> 
> In CONTRIBUTING.md:
> 
> > @@ -62,7 +62,16 @@ included in the project:
> >     git checkout -b <topic-branch-name>
> >     ```
> >  
> > -4. Install the [Jekyll](https://github.com/jekyll/jekyll/) gem to preview locally.
> > +4. Install the [Jekyll](https://github.com/jekyll/jekyll/) gem and dependencies to preview locally:
> > +
> > +    ```bash
> > +    # Install the needed gems
> > +    gem install jekyll jekyll-sitemap pygments.rb
> Better yet: gem install github-pages will install all gems which are available and potentially needed for a GH Pages build, including setting them to the correct versions for GH Pages.
> 
> —
> Reply to this email directly or view it on GitHub.
>